### DISCUSSION TITLE
Explnation plots and pool working

### DISCUSSION THREAD

**Discussion by casa10** [Chia-Network/chia-blockchain#7219] [Q&A]
Hye,
I don't understand well this way with portable plot NFT.

*** with the gui**
If you create a pot with interface gui and choose a pool (point "5 Join a pool" in the dashboard gui v1.2.0)..
1) After if you change of pool in the future, this plot will farm inside this new pool ?
2) what is the interest to complete the point "5 Join a pool" in the GUI dashboard if you can use your plots in others pools without needing replot plots for the others pools ?

*** with madmax**
3) How you use it with new plot ? 
Example :
chia_plot.exe -r 4 -f your_farmer_key -p your_pool_public_key -c address_of_the_pool_you_want_to_farm -t x:/ -2 y:/ -d Z:/Farm
4) with this option, if you change of pool after, these plots created with madmax and url_pool (option -c) will work in the new pool ?

Thank you for your help.

**Comment by loppefaaret** [comment]
what exactly are you wanting to plot with ? madmax ? 
whatever it is, if you are wanting to plot new pool switchable plots, you need to use the `-c` flag now, and stop using the `-p` flag!

**Reply by casa10** [reply]
Thank you for the part of answer about using option -c.
Do you know information about point 1) and 2) ?

**Reply by loppefaaret** [reply]
1. you are connecting plots to the plot NFT - that plot NFT you can point to self pooling, or any pool you want - and all the plots created for that plot NFT will farm under the selected pool
2. that is up to yourself to decide
